common/types/malfeasance.go

```diff
@@ -355,17 +350,19 @@ func MalfeasanceInfo(smesher NodeID, mp *MalfeasanceProof) string {
 			)
 		}
 	case InvalidPostIndex:
-		p, ok := mp.Proof.Data.(*InvalidPostIndexProof)
-		if ok {
-			p.Atx.Initialize()
-			b.WriteString(
-				fmt.Sprintf(
-					"cause: smesher published ATX %s with invalid post index %d in epoch %d\n",
-					p.Atx.ID().ShortString(),
-					p.InvalidIdx,
-					p.Atx.PublishEpoch,
-				))
-		}
+		// FIXME
+		// p, ok := mp.Proof.Data.(*wire.InvalidPostIndexProofV1)
+		// if ok {
+
+		// p.Atx.Initialize()
+		// b.WriteString(
+		// 	fmt.Sprintf(
+		// 		"cause: smesher published ATX %s with invalid post index %d in epoch %d\n",
+		// 		p.Atx.ID().ShortString(),
+		// 		p.InvalidIdx,
+		// 		p.Atx.PublishEpoch,
+		// 	))
+		// }
```

Comment:
Same as above: need to explain the problems encountered here. Also, is the situation with `InvalidPostIndex` just ignored here?

Reply:
Thanks, I forgot about it.